src/main/java/com/user/stock/service/StockService.java

```diff
@@ -0,0 +1,13 @@
+package com.user.stock.service;
+
+
+import com.user.stock.entity.Stocks;
+
+import java.util.List;
+
+public interface StockService {
+
+    List<Stocks> findAllStocks();
+
+    public Stocks findCertainStock(Integer symbol);
```

Comment:
こちらもCertainいらないかなと。
```suggestion
    public Stocks findStock(Integer symbol);
```
or
```suggestion
    public Stocks findStockBySymbol(Integer symbol);
```

Reply:
何度も大変申し訳ございません。
訂正いたしました。

2ecc34f

Reply:
OKです！